### ISSUE TITLE
Configuration file

### ISSUE THREAD

**Issue by schultyy** [CodePadawans/ataru#68]
As already suggested in #67, there should be a project related configuration file.
This file can also define some settings only related to a certain project like:
- MiniTest related settings like color, different reporter
- Location of setup file
- Files which should be checked automatically

**Comment by moonglum** [comment]
What format should that be? YAML? Ruby?

**Comment by schultyy** [comment]
Yaml is easy to process. JSON would also be fine. I would like to avoid XML ^^

**Comment by schultyy** [comment]
I would go for YAML

**Comment by moonglum** [comment]
:+1: for YAML :smile_cat: Then I vote for the file extension `.yaml` for that file.

**Comment by schultyy** [comment]
agreed :)

On 22 Sep 2014 at 17:12:50, Lucas Dohmen (notifications@github.com) wrote:

 for YAML  Then I vote for the file extension .yaml for that file.

**Comment by mmmries** [comment]
After trying out this gem in my gem I think it would be best to put any configuration in the <code>ataru_setup.rb</code> file. You could provide a template when you generate that file and make it work in a very normal ruby way. I'm thinking of something like:

``` ruby
Ataru.configure do |c|
  c.exclude_files = ["docs/future_improvements.md"]
end

require 'minitest-pride'
```

Just my 2 :euro:

**Comment by moonglum** [comment]
Yep, that is what we discussed in #75 as well :smile: We should update that here!
